pulsar-io/elastic-search/src/main/java/org/apache/pulsar/io/elasticsearch/client/RestClient.java

```diff
@@ -172,6 +173,9 @@ private SSLContext buildSslContext(ElasticSearchConfig config) throws NoSuchAlgo
                 sslContextBuilder.loadTrustMaterial(new File(sslConfig.getTruststorePath()),
                         sslConfig.getTruststorePassword().toCharArray());
             }
+            if (sslConfig.isDisableCertificateValidation()) {
+                sslContextBuilder.loadTrustMaterial(null, TrustAllStrategy.INSTANCE);
```

Comment:
Please add logging at warning level when certification validation is disabled. Please address this for hostnameVerification too if there's no logging currently.